### PR TITLE
Feature/case-studies/breadcrumbs

### PR DESCRIPTION
**Pull request by vicarali**
Change the static Industry tag on Case Study pages for the same breadcrumbs menu used on the Industry pages

### CODE REVIEW

File: version_control/Codurance_September2020/css/_menus.css [hunk 1]
```diff
@@ -32,6 +32,10 @@ menu items appear, including the link and hover effects.
 }
 
 /* Breadcrumbs */
+.hs-breadcrumb-menu-wrapper {
```
Comment (RodrigoCastroS) [inline version_control/Codurance_September2020/css/_menus.css:35]: Maybe here we can use a spacing variable instead a fix pixel number. What do you think?

File: version_control/Codurance_September2020/templates/case-study-template.html [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
         Case Study
       </span>
-      <p class="case-study-header__industry">
-        {% module "industry_text"
-          label="Industry Text",
-          path="@hubspot/text",
-          value="Health Sector"
-          no_wrapper=True
+
+      <div class="case-study-header__breadcrumbs">
```
Comment (RodrigoCastroS) [inline version_control/Codurance_September2020/templates/case-study-template.html:22]: I'm not really sure how Hubspot implements breadcrumbs but we could take in consideration accessibility. This is a [link to WAI standards](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/)

**Review by RodrigoCastroS — DISMISSED** [review]
I've left you a couple of comments. We can discuss it further if you want to improve what we have. Other than that, well done! 🚀

File: version_control/Codurance_September2020/css/_menus.css (hunk 1 above)
Comment (vicarali) [reply version_control/Codurance_September2020/css/_menus.css:35]: I wasn't sure with doing that due to this variables being mainly to define spaces between elements, but I guess it does make sense to use it here, so I just did the change!

File: version_control/Codurance_September2020/templates/case-study-template.html (hunk 2 above)
Comment (vicarali) [reply version_control/Codurance_September2020/templates/case-study-template.html:22]: Hubspot uses a div wrapper with `role="navigation"`, perhaps it is enough?

**Pull request merged**